Adding TF2 timer interface migration information (#462)
ros2/ros2_documentation: source/Releases/Release-Eloquent-Elusor.rst

`https://github.com/ros2/rviz/pull/455 <https://github.com/ros2/rviz/pull/455>`_

TF2 Buffer
^^^^^^^^^^

> TF2 buffers have to be given a timer interface, ei

Reviewer: ```suggestion
TF2 buffers now have to be given a timer interface, e.g.:
```

.. code-block:: cpp

    tf_ = std::make_shared<tf2_ros::Buffer>(get_clock());

>     /* New addition */

Reviewer: ```suggestion
    // The next two lines are new in Eloquent
```

    auto timer_interface = std::make_shared<tf2_ros::CreateTimerROS>(
      this->get_node_base_interface(),
      this->get_node_timers_interface());
    tf_->setCreateTimerInterface(timer_interface);

>     transform_listener_ = std::make_shared<tf2_ros::TransformListener>(*tf_);

Reviewer: And then a comment above here saying something like:

```
// Pass the tf2_ros::Buffer to the TransformListener as before
```

`https://github.com/ros2/rviz/pull/455 <https://github.com/ros2/rviz/pull/455>`_

TF2 Buffer
^^^^^^^^^^

> TF2 buffers now have to be given a timer interface. If a timer interface is not given, an exception will be thrown.

Reviewer 2: One sentence per line.
```suggestion
TF2 buffers now have to be given a timer interface.
If a timer interface is not given, an exception will be thrown.
```

Author: done!
